atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1545,6 +1548,11 @@ void WebContents::DestroyWebContents() {
   }
 }
 
+void WebContents::OnSafeBrowsingHit(
+  const security_interstitials::UnsafeResource& resource) {
+  Emit("safebrowsing-hit");
```

Comment:
don't we need to pass the details along? For instance this might be a subresource load which we would handle differently than a main frame load